src/relay/qnn/util.cc

```diff
@@ -22,13 +22,49 @@
  * \brief Utility functions for QNN.
  */
 
+#include <limits>
+
 #include "util.h"
 #include "../pass/pattern_util.h"
 
 namespace tvm {
 namespace relay {
 namespace qnn {
 
+/* \brief This function implements the rounding part of ARMv7 NEON VQRDMULH
+ * instruction. For code reuse, the multiplied tensor is directly passed in
+ * as parameter. Reference implementation:
+ * https://github.com/google/gemmlowp/blob/2483d846ad865dd4190fe4a1a1ba2d9cfcea78e1/fixedpoint/fixedpoint.h#L337
```

Comment:
What is the license of GEMMLOWP? If you understand the algorithm and the code is written by yourself, you could remove this link so that we could avoid the license problem.

Reply:
I assume a reference link will not trigger license problem? The code is reimplemented in relay, my intension to add this reference link is to mark the origin of this implementation so that other developers would get better idea what we are doing here.

Reply:
We would be better remove this so that we won't worry about it. The same thing is happened in other PRs too. We all suggest authors removing these links. Thanks.

Reply:
Sure, I'll respect that